### ISSUE TITLE
Duplicate metrics are being exposed

### ISSUE THREAD

**Issue by themkarimi** [Lightstreamer/Lightstreamer-utility-MetricsExporter-Adapter#3]
We have been receiving lots of warnings and alerts from our Prometheus instance regarding the fact that Lightstreamer is exposing a lot of duplicate metrics, at first we thought there might be a configuration issue but then after reviewing the metrics exposed by the Lightstreamer we saw that it's right, for example lightstreamer_AdapterSet_CurrentSessions metrics is exposed twice among other metrics, this has not been an issue before but since we upgraded our Prometheus instance to the version v2.52.0 it seems that Prometheus is being strict regarding the duplicate metrics. 
It would be awesome if you could provide a fix for this issue.
Many Thanks

**Comment by Bravo27** [comment]
Thank you for reporting the issue, but unfortunately, I am unable to replicate the problem.

For example, considering the metric you mentioned, **lightstreamer_AdapterSet_CurrentSessions**, here is what I see when querying the metrics exporter directly on port 9876:

```
# HELP lightstreamer_AdapterSet_CurrentSessions The number of current client Sessions bound to this Adapter Set. The count includes "prestarted" sessions.
# TYPE lightstreamer_AdapterSet_CurrentSessions gauge
lightstreamer_AdapterSet_CurrentSessions{AdapterSetName="METRICS",} 0.0
lightstreamer_AdapterSet_CurrentSessions{AdapterSetName="WELCOME",} 2.0
```
In this case, there is no duplication; it correctly reports the number of active sessions for each installed adapter set. In my case, there are two ("METRICS" and "WELCOME").

Could you please provide more details or examples of the output you are getting? This would help us to understand and address the issue more effectively.

**Comment by themkarimi** [comment]
Thanks for checking this out. I have attached the output of our instance metrics endpoint here. 
[metrics.txt](https://github.com/user-attachments/files/15913786/metrics.txt)
see the lines 226 and 486 as an example.

**Comment by Bravo27** [comment]
Published release v0.2.0; should fix the issue.

**Comment by themkarimi** [comment]
Many thanks. We are gonna test this out and if the issue was fixed I will close this case.
